content/docs/developing-operators.md

```diff
@@ -71,19 +76,27 @@ spec:
 This is a pretty normal Kubernetes YAML file defining a deployment. However, you can already see the KUDO templating language in action on the line referencing `.Params.Replicas`. This will get substituted during installation by merging what is in `params.yaml` and overrides defined before install. So let’s define the last missing piece, `params.yaml`.
```

Comment:
I think would be great also add the steps to make clear that they should create the file in the root.